server/db/interface.js

```diff
@@ -0,0 +1,76 @@
+var Sequelize = require('sequelize');
+
+var config = require('./postgres.config.js');
+var url = process.env.DATABASE_URL; //SET THIS UP PROPERLY SOON
+
+var listItem = require('./models/ListItem.js');
+var household = require('./models/Household.js');
+var reckoning = require('./models/Reckoning.js');
+var user = require('./models/User.js');
+
+var dbEnvironment = process.env.NODE_ENV;
+
+var schema;
+var shouldForce;
+
+if (dbEnvironment === 'test') {
+  schema = config.testSchema;
+} else {
+  schema = config.mainSchema;
+}
+
+//not sure what the ssl does yet, will look this up in a minute
+//yell at Naomi if she forgets to get back to this
+var db = new Sequelize(url, {ssl: true, {schema: schema}});
```

Comment:
Sova added the ssl option because of his Linux or something. Normally, setting environment variable `PGSSLMODE` to `require` does enough of the SSL configuration, I think